packages/flutter_tools/lib/src/protocol_discovery.dart

```diff
@@ -39,9 +40,11 @@ class ProtocolDiscovery {
     required Logger logger,
   }) {
     const String kObservatoryService = 'Observatory';
+    const String kServicePrefixRegExp = '(?:Observatory|The Dart VM Service is)';
```

Comment:
can this be a static or global and shared between other diffs in this PR?

Reply:
Is there a constants file that this could live in?

Reply:
Done.